apps/boilerplate-front-end/src/components/federatedSearch/SCSS/federatedSearch.scss

```diff
@@ -189,22 +189,33 @@
   &__search {
     display: flex;
     align-items: center;
+    margin-top: 0.5rem;
+    p {
+      font-weight: 300;
+      font-size: 0.8rem;
```

Comment:
Put the Font size and weight without partial because of the Hove I didn't want on the query